src/content/en/updates/posts/2016/04/ump.markdown

```diff
@@ -0,0 +1,156 @@
+
+<a href="https://simpl.info/video/offline"><img class="screenshot-portrait" src="/web/updates/images/2016/04/ump/screenshot-file.jpg" alt="Screenshot of playback in Chrome on Android of video stored using the File APIs"></a>
+
+{% highlight javascript %}
+function writeToFile(fileEntry, blob) {
+  fileEntry.createWriter(function(fileWriter) {
+    fileWriter.onwriteend = function() {
+      readFromFile(fileEntry.fullPath);
+    };
+    fileWriter.onerror = function(e) {
+      log('Write failed: ' + e.toString());
+    };
+    fileWriter.write(blob);
+  }, handleError);
+}
+
+function readFromFile(fullPath) {
+  window.fileSystem.root.getFile(fullPath, {}, function(fileEntry) {
+    fileEntry.file(function(file) {
+      var reader = new FileReader();
+      reader.onloadend = function() {
+        video.src = URL.createObjectURL(new Blob([this.result]));
+      };
+      reader.readAsArrayBuffer(file);
+    }, handleError);
+  }, handleError);
+}
+{% endhighlight %}
+
+The Unified Media Pipeline has also been [enabled for MSE and EME](https://groups.google.com/a/chromium.org/forum/#!topic/chromium-reviews/Qi4dLcKjcCM).
+
+This is another step towards unifying mobile and desktop Chrome. You don't need to change your code, but building a consistent media experience across desktop and mobile should now be easier, since the media stack is the same across platforms. Debugging with Chrome DevTools? Mobile emulation now uses the 'real' audio and video stack.
+
+If you experience problems as a result of the Unified Media Pipeline, please file issues on the [implementation bug](https://groups.google.com/a/chromium.org/forum/#!topic/chromium-reviews/Qi4dLcKjcCM) or via [new.crbug.com](https://new.crbug.com).
+
+## Demos
+
+* <a href="https://googlechrome.github.io/samples/service-worker/prefetch-video/index.html">Cache video with a service worker</a>
+* <a href="https://simpl.info/video/playbackrate">Media `playbackRate`</a>
+* <a href="https://simpl.info/mediarecorder">MediaRecorder: playback using a Blob URL</a>
+* <a href="https://simpl.info/video/offline">Offline video implemented with the File APIs</a>
+
+## Relevant bugs
+
+* [Tracking issues for the Unified Media Pipeline](https://bugs.chromium.org/p/chromium/issues/detail?id=507834)
+* [UMP Field Trial](https://groups.google.com/a/chromium.org/forum/#!topic/chromium-reviews/okUkrNc0z6w)
+* [MSE, EME and the Unified Media Pipeline](https://groups.google.com/a/chromium.org/forum/#!topic/chromium-reviews/Qi4dLcKjcCM)
+
+There are a couple of bugs affecting &lt;video&gt;, service workers and the Cache Storage API:
+
+* [&lt;video&gt; triggers a mode: cors request and won't accept an opaque service worker response](https://bugs.chromium.org/p/chromium/issues/detail?id=546076)
+* [Seeking doesn't work in videos served by service worker cache](https://bugs.chromium.org/p/chromium/issues/detail?id=575357)
+
+
+## Browser support
+
+* Behind a flag in Chrome Dev since January 2016
```

Comment:
UMP has just been pushed back, so I think enabled by default from 52 is correct now.